### DISCUSSION TITLE
What is the meaning of star at the end?

### DISCUSSION THREAD

**Discussion by vasnt** [khoj-ai/khoj#115] [Q&A] [answered]
Local: Your personal data stays local. All search, indexing is done on your machine[*](https://github.com/debanjum/khoj#miscellaneous)

**Comment by debanjum** [comment, marked as answer]
Hi @vasnt, thanks on opening a discussion topic instead of an issue for this 🙂

The asterisk is for this warning/disclaimer:
If you wish to try the /api/beta/{chat,search} API you need to set your [OpenAI API](https://openai.com/api/) key in Khoj. In that case, your query/top result is sent to OpenAI for processing (e.g summarization, content-type categorization). But this requires explicit user buy-in

If you click the asterisk, it'll lead you to the Miscellaneous section of the readme at https://github.com/debanjum/khoj#miscellaneous, which says something similar

Let me know if that clarifies what the star is about?

**Reply by vasnt** [reply]
that is crystal clear, thanks.

**Reply by debanjum** [reply]
Brilliant!
